--- rc.S	2018-11-26 16:17:38.155343829 -0700
+++ rc.S.new	2018-11-26 16:17:41.095394987 -0700
@@ -322,23 +322,6 @@
   /etc/rc.d/rc.cgred start
 fi
 
-# Clean up some temporary files:
-rm -f /var/run/* /var/run/*/* /var/run/*/*/* /etc/nologin \
-  /etc/dhcpc/*.pid /etc/forcefsck /etc/fastboot \
-  /var/state/saslauthd/saslauthd.pid \
-  /tmp/.Xauth* 1> /dev/null 2> /dev/null
-if [ -d /var/lib/pkgtools/setup/tmp ]; then
-  ( cd /var/lib/pkgtools/setup/tmp && rm -rf * )
-elif [ -d /var/log/setup/tmp ]; then
-  ( cd /var/log/setup/tmp && rm -rf * )
-fi
-( cd /tmp && rm -rf kde-[a-zA-Z]* ksocket-[a-zA-Z]* hsperfdata_[a-zA-Z]* plugtmp* )
-
-# Clear /var/lock/subsys:
-if [ -d /var/lock/subsys ]; then
-  rm -f /var/lock/subsys/*
-fi
-
 # Create /tmp/{.ICE-unix,.X11-unix} if they are not present:
 if [ ! -e /tmp/.ICE-unix ]; then
   mkdir -p /tmp/.ICE-unix
