--- rc.M	2021-11-12 11:52:01.000000000 -0700
+++ rc.M.new	2022-03-14 13:11:39.692556035 -0600
@@ -88,6 +88,16 @@
   /etc/rc.d/rc.syslog start
 fi
 
+### RL Added ###
+# Make sure X knows about all the installed fonts
+if [ -x /usr/bin/mkfontdir -a -x /usr/bin/mkfontscale ]; then
+  for FDIR in /usr/share/fonts/*; do
+    /usr/bin/mkfontscale ${FDIR}
+    /usr/bin/mkfontdir ${FDIR}
+  done
+fi
+### RL Added ###
+
 # Update the X font indexes:
 if [ -x /usr/bin/fc-cache ]; then
   echo "Updating X font indexes:  /usr/bin/fc-cache -f &"
