--- rc.modules	2021-04-23 12:31:34.000000000 -0600
+++ rc.modules.new	2023-09-09 03:30:11.934380947 -0600
@@ -5,6 +5,9 @@
 # Determine the version of the running kernel:
 RELEASE=$(uname -r)
 
+# RL Added: Ensure that we run depmod --all during startup
+rm -f /lib/modules/${RELEASE}/modules.{dep,dep.bin}
+
 # Update kernel module dependencies:
 if [ -e "/lib/modules/$RELEASE/modules.dep" ]; then 
   echo "Updating module dependency list for $RELEASE:  /sbin/depmod --quick"
