--- rc.inet2	2019-10-13 12:53:50.000000000 -0600
+++ rc.inet2.new	2019-12-27 08:08:16.771994421 -0700
@@ -110,7 +110,7 @@
 # Start the OpenSSH SSH daemon:
 if [ -x /etc/rc.d/rc.sshd ]; then
   echo "Starting OpenSSH SSH daemon:  /usr/sbin/sshd"
-  /etc/rc.d/rc.sshd start
+  /etc/rc.d/rc.sshd start 1>/dev/null &
 fi
 
 # Start the BIND name server daemon:
@@ -137,6 +137,18 @@
   /etc/rc.d/rc.nfsd start
 fi
 
+### RL Added ###
+# Start the IPv6 radvd service
+if [ -x /etc/rc.d/rc.radvd ]; then
+  /etc/rc.d/rc.radvd start
+fi
+
+# Start the libvirt daemon
+if [ -x /etc/rc.d/rc.libvirtd ]; then
+  /etc/rc.d/rc.libvirtd start
+fi
+### RL Added ###
+
 # Stuff you won't need follows.  ;-)
 
 # # Start the network routing daemon:
