--- rc.M	2024-06-28 19:01:13.000000000 +0000
+++ rc.M.new	2024-10-23 08:57:43.374531409 +0000
@@ -100,6 +100,16 @@
   /etc/rc.d/rc.iceccd start
 fi
 
+### RL Added ###
+# Make sure X knows about all the installed fonts
+if [ -x /usr/bin/mkfontdir -a -x /usr/bin/mkfontscale ]; then
+  for FDIR in /usr/share/fonts/*; do
+    /usr/bin/mkfontscale ${FDIR}
+    /usr/bin/mkfontdir ${FDIR}
+  done
+fi
+### RL Added ###
+
 # Update the X font indexes:
 if [ -x /usr/bin/fc-cache ]; then
   echo "Updating X font indexes:  /usr/bin/fc-cache -f &"
