--- rc.M	2021-01-20 12:23:16.446771722 -0700
+++ rc.M.new	2021-01-20 12:23:23.556804923 -0700
@@ -411,6 +411,34 @@
   /etc/rc.d/rc.iceccd start
 fi
 
+### RL Added ###
+# This doesnt seem to be done anywhere else
+if [ -x /etc/rc.d/rc.rl_certupdate ]; then
+  /etc/rc.d/rc.rl_certupdate
+fi
+
+# Setup and Start docker
+if [ -x /etc/rc.d/rc.rl_docker ]; then
+  /etc/rc.d/rc.rl_docker setup
+  /etc/rc.d/rc.rl_docker start
+fi
+
+# Start rasdaemon:
+if [ -x /etc/rc.d/rc.rasdaemon ]; then
+  /etc/rc.d/rc.rasdaemon start
+fi
+
+# Start Lighttpd web server:
+if [ -x /etc/rc.d/rc.lighttpd ] && [ -r /etc/lighttpd/lighttpd.conf ]; then
+  /etc/rc.d/rc.lighttpd start
+fi
+
+# Install VirtualBox if available
+if [ -x /etc/rc.d/rc.VirtualBox ]; then
+  /etc/rc.d/rc.VirtualBox
+fi
+### RL Added ###
+
 # If there are SystemV init scripts for this runlevel, run them.
 if [ -x /etc/rc.d/rc.sysvinit ]; then
   /etc/rc.d/rc.sysvinit
