--- rc.modules	2015-12-16 04:10:05.000000000 +0000
+++ rc.modules.new	2017-11-20 18:43:48.255270984 +0000
@@ -5,15 +5,6 @@
 # Determine the version of the running kernel:
 RELEASE=$(uname -r)
 
-# Update kernel module dependencies:
-if [ -e "/lib/modules/$RELEASE/modules.dep" ]; then 
-  echo "Updating module dependency list for $RELEASE:  /sbin/depmod --quick"
-  /sbin/depmod --quick
-else 
-  echo "Creating module dependency list for $RELEASE:  /sbin/depmod --all"
-  /sbin/depmod --all
-fi
-
 # Run any rc.modules-$(uname -r) file that exists (this is used
 # if you have specific modules which should only be loaded for
 # specific kernel versions):
