--- rc.S	2018-11-26 16:16:09.563525376 -0700
+++ rc.S.new	2018-11-26 16:16:15.383643966 -0700
@@ -279,12 +279,6 @@
   /sbin/sysctl -e --system 2> /dev/null | grep -v "Applying /etc/sysctl.conf"
 fi
 
-# Check all the non-root filesystems:
-if [ ! -r /etc/fastboot ]; then
-  echo "Checking non-root filesystems:"
-  /sbin/fsck $FORCEFSCK -C -R -A -a
-fi
-
 # Mount usbfs only if it is found in /etc/fstab:
 if grep -wq usbfs /proc/filesystems; then
   if ! grep -wq usbfs /proc/mounts ; then
