--- rc.udev	2017-04-23 19:50:22.000000000 +0000
+++ rc.udev.new	2017-11-14 15:12:42.714090847 +0000
@@ -7,7 +7,7 @@
 
 check_mounted() {
   grep -E -q "^[^[:space:]]+ $1 $2" /proc/mounts
-  return $?	
+  return $?
 }
 
 mount_devpts() {
@@ -20,7 +20,7 @@
 mount_devshm() {
   if ! check_mounted /dev/shm tmpfs ; then
     mkdir /dev/shm 2> /dev/null
-    mount /dev/shm
+    /sbin/mount shm /dev/shm -t tmpfs -o nodev,nosuid,noexec,noatime,nodiratime
   fi
 }
 
