--- rc.M	2018-08-13 15:39:33.000000000 -0600
+++ rc.M.new	2018-12-31 07:38:06.725867723 -0700
@@ -77,6 +77,15 @@
   /etc/rc.d/rc.syslog start
 fi
 
+### RL Added ###
+# Make sure X knows about all the installed fonts
+if [ -x /usr/bin/mkfontdir ]; then
+  for FDIR in /usr/share/fonts/*; do
+    mkfontdir ${FDIR}
+  done
+fi
+### RL Added ###
+
 # Update the X font indexes:
 if [ -x /usr/bin/fc-cache ]; then
   echo "Updating X font indexes:  /usr/bin/fc-cache -f &"
