--- rc.M	2018-07-10 04:51:43.525115311 -0600
+++ rc.M.new	2018-07-10 04:53:48.240451322 -0600
@@ -120,15 +120,6 @@
   /etc/rc.d/rc.ntpd start
 fi
 
-# Remove stale locks and junk files (must be done after mount -a!)
-/bin/rm -f /var/lock/* /var/spool/uucp/LCK..* /tmp/.X*lock /tmp/.X11-unix/* 2> /dev/null
-
-# Remove stale hunt sockets so the game can start.
-if [ -r /tmp/hunt -o -r /tmp/hunt.stats ]; then
-  echo "Removing your stale hunt sockets from /tmp."
-  /bin/rm -f /tmp/hunt*
-fi
-
 # Ensure basic filesystem permissions sanity.
 chmod 755 / 2> /dev/null
 chmod 1777 /tmp /var/tmp
