--- rc.M	2017-11-20 15:12:56.313998119 +0000
+++ rc.M.new	2017-11-20 15:14:01.113331557 +0000
@@ -54,12 +54,6 @@
   /etc/rc.d/rc.syslog start
 fi
 
-# Update the X font indexes:
-if [ -x /usr/bin/fc-cache ]; then
-  echo "Updating X font indexes:  /usr/bin/fc-cache -f &"
-  /usr/bin/fc-cache -f &
-fi
-
 # Run rc.udev again.  This will start udev if it is not already running
 # (for example, upon return from runlevel 1), otherwise it will trigger it
 # to look for device changes and to generate persistent rules if needed.
