--- rc.S	2018-02-06 16:21:50.473657892 +0000
+++ rc.S.new	2018-02-06 16:22:19.213370852 +0000
@@ -300,7 +300,7 @@
   fi
 fi
 
-# Mount non-root file systems in fstab, but not NFS or SMB 
+# Mount non-root file systems in fstab, but not NFS or SMB
 # because TCP/IP is not yet configured, and not proc or sysfs
 # because those have already been mounted.  Also check that
 # devpts is not already mounted before attempting to mount
@@ -308,15 +308,15 @@
 # We also need to wait a little bit to let USB and other
 # hotplugged devices settle (sorry to slow down the boot):
 echo "Mounting non-root local filesystems:"
-sleep 3
+#sleep 3 - RL ADDED - I dont want to wait
+FSIGNORE="nonfs,nosmbfs,nocifs,noproc,nosysfs,noslaxdrive"
 if /bin/grep -wq devpts /proc/mounts ; then
-  # This pipe after the mount command is just to convert the new
-  # mount verbose output back to the old format that contained
-  # more useful information:
-  /sbin/mount -a -v -t nonfs,nosmbfs,nocifs,noproc,nosysfs,nodevpts | grep successfully | cut -f 1 -d : | tr -d ' ' | while read dev ; do mount | grep " ${dev} " ; done
-else
-  /sbin/mount -a -v -t nonfs,nosmbfs,nocifs,noproc,nosysfs | grep successfully | cut -f 1 -d : | tr -d ' ' | while read dev ; do mount | grep " ${dev} " ; done
+  FSIGNORE+=",nodevpts"
 fi
+# This pipe after the mount command is just to convert the new
+# mount verbose output back to the old format that contained
+# more useful information:
+/sbin/mount -a -v -t $FSIGNORE | grep successfully | cut -f 1 -d : | tr -d ' ' | while read dev ; do mount | grep " ${dev} " ; done
 
 # Enable swapping again.  This is needed in case a swapfile is used,
 # as it can't be enabled until the filesystem it resides on has been
