--- rc.M	2020-08-17 02:45:13.035926229 -0600
+++ rc.M.new	2020-08-17 02:51:08.557385800 -0600
@@ -409,6 +409,34 @@
   /etc/rc.d/rc.iceccd start
 fi
 
+### RL Added ###
+# This doesnt seem to be done anywhere else
+if [ -x /usr/sbin/update-ca-certificates ]; then
+  /usr/sbin/update-ca-certificates -f
+fi
+
+# Setup and Start docker
+if [ -x /etc/rc.d/rc.rl_docker ]; then
+  /etc/rc.d/rc.rl_docker setup
+  /etc/rc.d/rc.rl_docker start
+fi
+
+# Start rasdaemon:
+if [ -x /etc/rc.d/rc.rasdaemon ]; then
+  /etc/rc.d/rc.rasdaemon start
+fi
+
+# Start Lighttpd web server:
+if [ -x /etc/rc.d/rc.lighttpd ] && [ -r /etc/lighttpd/lighttpd.conf ]; then
+  /etc/rc.d/rc.lighttpd start
+fi
+
+# Install VirtualBox if available
+if [ -x /etc/rc.d/rc.VirtualBox ]; then
+  /etc/rc.d/rc.VirtualBox
+fi
+### RL Added ###
+
 # If there are SystemV init scripts for this runlevel, run them.
 if [ -x /etc/rc.d/rc.sysvinit ]; then
   /etc/rc.d/rc.sysvinit
